ui/app/helpers/clean-keycommand.js

```diff
@@ -0,0 +1,23 @@
+// @ts-check
+import { helper } from '@ember/component/helper';
+
+const KEY_ALIAS_MAP = {
+  ArrowRight: '→',
+  ArrowLeft: '←',
+  ArrowUp: '↑',
+  ArrowDown: '↓',
+  '+': ' + ',
+  // Enter: '⏎',
+  // Tab: '⇥',
+  // Space: '␣',
+  // Shift: '⇧',
+};
+
+export default helper(function cleanKeycommand([part] /*, named*/) {
```

Comment:
`question`:  Why do we pass in a `key` in the template, but we name the argument `part`?

Reply:
The key might be a sequence, like `["g", "v"]` for `go to variables`. `g` and `v` would be individual `part`s.

Reply:
Hm you know what? You bring up a good point here. I'm using too many words to mean roughly the same thing. Will bring them all down to `part`.